presto-product-tests/src/main/java/io/prestosql/tests/cli/TestPrestoCli.java

```diff
@@ -122,7 +122,7 @@ public void shouldRunQuery()
     public void shouldRunBatchQuery()
             throws Exception
     {
-        launchPrestoCliWithServerArgument("--execute", "select * from hive.default.nation;");
+        launchPrestoCliWithServerArgument("--execute", escapeForBash("select * from hive.default.nation;"));
```

Comment:
Any reason why escaping is not a hidden aspect of the `launchPrestoCliWithServerArgument` method? or even deeper?

Reply:
I could do that

Reply:
But I would need separate methods that test redirects

Reply:
If we really have to use bash wrapper around CLI, let's do that in an isolated place, single test method.
This way:

- we do not need to call `escapeForBash` (which we will be forgetting to call; and we do not forget, this reduces code readability)
- we can remove bash more easily, which is my ultimate goal :)